README.md

```diff
@@ -31,14 +31,15 @@
 - HTML
   -CSS
   -FLEXBOX
+  -GRID
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
 - **[Toolbar and Headline section]**
-- **[Header]**
-- **[Image]**
+- **[Header ]**
+- **[WORK SECTION]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
### UPDATE THE FOLLOWING SECTIONS OF THE README TO MAKE IT MORE PROFESSIONAL


#### REMOVE ALL THE FOLLOWING PLACEHOLDER TEXT

- [ ] Placeholder text guides to help you add the appropriate content to its respective sections and are not expected to be in your readme after making the update to suit your project needs. let's remove them.

![Screenshot from 2023-02-22 16-14-10](https://user-images.githubusercontent.com/59397159/220666754-ebc1a7e4-d053-41b0-99d3-5bc37dc7f05c.png)
![Screenshot from 2023-02-22 16-14-43](https://user-images.githubusercontent.com/59397159/220666762-1a901df3-d433-439f-8131-8432f2c8641b.png)
![Screenshot from 2023-02-22 16-14-51](https://user-images.githubusercontent.com/59397159/220666768-255eeff6-9a79-44e9-9b11-ec19e6bae292.png)


###  Show your support

- [ ] Please, kindly improve the text to `If you like this project, give it a` :star: to make it more descriptive.

*improve text below* :point_down: 
![Screenshot from 2023-02-22 16-19-51](https://user-images.githubusercontent.com/59397159/220667934-b9a9bc50-f85b-43f9-a88b-54ff2918ab37.png)


### Acknowledgments

- [ ] Let's also improve the text to be more descriptive. e.g giving acknowledgment to Figma and microverse below
:point_right:   *Template from this [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=N59QdTSJoDabEoA7-0) by [Microverse](https://bit.ly/MicroverseTN)* 
See the image below:

*text to be improved* :point_down: 
![Screenshot from 2023-02-22 16-27-00](https://user-images.githubusercontent.com/59397159/220669902-d6147bd2-bbc2-44a3-9e42-a1ab45e54d73.png)